static/js/publisher/release/components/releasesTableChannelCell.js

```diff
@@ -0,0 +1,407 @@
+import React, { Fragment } from "react";
```

Comment:
This file is mostly extracted functionality that was previously part of ReleasesTableRow component.